contractcourt/channel_arbitrator.go

```diff
@@ -811,11 +811,8 @@ func (c *ChannelArbitrator) relaunchResolvers(commitSet *CommitSet,
 
 // Report returns htlc reports for the active resolvers.
 func (c *ChannelArbitrator) Report() []*ContractReport {
-	c.activeResolversLock.RLock()
-	defer c.activeResolversLock.RUnlock()
-
 	var reports []*ContractReport
-	for _, resolver := range c.activeResolvers {
+	for _, resolver := range c.resolvers() {
```

Comment:
Where was the race? At a glance, everything looks to be mutex proteted.

Reply:
hmm looks like it no longer existed due to the fix of `resolved` and `launched`, now removed.